conf/application.conf

```diff
@@ -94,12 +94,12 @@ contact-frontend {
 }
 
 timeout-dialog {
-  timeout   = 900
-  countdown = 120
+  timeout   = 9
```

Comment:
@BriWak Is it correct, shouldn't be timeout:900, countdown:400 and 
session {
   timeoutSeconds = 900
 }